src/services/auth.service.ts

```diff
@@ -0,0 +1,52 @@
+declare const gapi: any;
+
+export class AuthService {
+
+  public loadGapi(): void {
+    // Get the gapi script from Google.
+    const gapiScript = document.createElement('script');
```

Comment:
How will this work on Leo where gapi has already been loaded and may be mid-auth

Reply:
So my hope was that all of this would be harmless if gapi is already loaded. But I haven't had a chance yet to test this out in Leo... for now, just focusing on getting things working locally.

If we think it is going to be a problem, we could have a config setting that determines whether or not to include this, and only set it to true in the local environment?

Reply:
Fine to leave this for now until we have the capability of testing this on Leo.